examples/platforms/nrf52840/platform-nrf5.h

```diff
@@ -175,4 +175,12 @@ otError nrf5SdErrorToOtError(uint32_t aSdError);
 void nrf5SdSocFlashProcess(uint32_t aEvtId);
 #endif // SOFTDEVICE_PRESENT
 
+#define US_PER_MS            1000ULL
+
+/**
+ * Function for geting current time in mircosecond.
+ *
+ */
+uint64_t AlarmGetCurrentTime();
```

Comment:
nit: What about renaming it to nrf5AlarmGetCurrentTime to follow the convention above? Then you can move it up to the nrf5Alarm* API.